ViewModel/ImpactAreaScenario/Results/DamageWithUncertaintyVM.cs

```diff
@@ -16,44 +16,38 @@ public class DamageWithUncertaintyVM : BaseViewModel
 
         public List<EadRowItem> Rows { get; } = new List<EadRowItem>();
         public double Mean { get; set; }
-        public DamageWithUncertaintyVM(ImpactAreaScenarioResults iasResult)
+        public DamageWithUncertaintyVM(ImpactAreaScenarioResults iasResult, ScenarioResults scenarioResults)
         {
             int impactAreaID = iasResult.ImpactAreaID;
             Mean = iasResult.MeanExpectedAnnualConsequences(impactAreaID: impactAreaID);
             IHistogram totalHistogram = iasResult.ConsequenceResults.GetConsequenceResultsHistogram(impactAreaID: impactAreaID);
-            double[] binsAsDoubles = totalHistogram.BinCounts.Select(x => (double)x).ToArray();
+            double[] binsAsDoubles = totalHistogram.BinCounts.Select(x => (double)x/totalHistogram.SampleSize).ToArray();
```

Comment:
It's not very obvious what's going on here.  The API should be doing this kind of logic, not you, why are we dividing it by the sample size?

Reply:
github issue made.